clar2wasm/src/deserialize.rs

```diff
@@ -843,15 +869,15 @@ impl WasmGenerator {
         let result_offset = self.module.locals.add(ValType::I32);
         let element_size = get_type_size(element_ty);
         block
-            .global_get(self.stack_pointer)
+            .local_get(offset_result)
```

Comment:
We have two variables `offset_result` and `result_offset`. That seems confusing...

Reply:
You're right, renamed it